relay-server/src/actors/global_config.rs

```diff
@@ -231,9 +258,12 @@ impl GlobalConfigService {
                 let mut success = false;
                 match config.global {
                     Some(global_config) => {
-                        // Notifying subscribers only fails when there are no
-                        // subscribers.
```

Comment:
I find this comment useful and I'd suggest restoring it.

Reply:
we now use send_replace, which never fails, so the old doc is outdated